js/api-augment/definitions/messages.ts

```diff
@@ -24,6 +24,7 @@ export default {
     },
     MessageResponse: {
       payload: "Vec<u8>", //  Serialized data in a user-defined schema format
+      cid: "Vec<u8>", // The content address for an IPFS payload
```

Comment:
I don't think this matches now right? Several of these should be `Option`?

Also don't forget to run `make js` to update the other parts of the metadata off of this.